app/javascript/articles/LoadingArticle.jsx

```diff
@@ -11,7 +11,7 @@ export const LoadingArticle = ({ version }) => {
     </div>
   );
   return (
-    <div className="crayons-story">
+    <div className="crayons-story" title="Loading posts...">
```

Comment:
Let's an user requiring a11y know what's going on. It also allows us to use preact-testing-library's `getByTitle`.